docs/smart_scripts.md

```diff
@@ -47,64 +49,66 @@
 [8]: #event_flags
 [9]: #action_type
 [10]: #target_type
-[11]: #comment
+[11]: #target_x
+[12]: #target_y
+[13]: #target_z
+[14]: #target_o
+[15]: #comment
 
 **Description of the fields**
 
 ### entryorguid
 
--   EntryOrGuid &gt; 0: entry of the creature / game object / etc.
+- EntryOrGuid > 0: `entry` of the creature / game object / etc.
+- EntryOrGuid < 0: `guid` of the creature / game object / etc.
 
-<!-- -->
+**Depends on source\_type.**
 
--   EntryOrGuid &lt; 0: guid of the creature / game object / etc.
+> In other words:
 
-<!-- -->
+> If the entryorguid, is greater than 0, we would be using the `entry` of a creature, in other words, we are applying the fix, to a whole family of npcs, objects... which, while using all the same `entry`, all run the same script.
 
--   Depends on source\_type.
+> On the other hand, if the entryorguid is less than 0, we would use the `guid` of the creature table, being able to apply the fix, only to 1 creature, of a group of many others. But maybe we don't want all of them to behave the same way.
```

Comment:
Line 67 - 71 is just overexplaining.
in my opinion

```
- EntryOrGuid > 0: `entry` of the creature / game object / etc.
- EntryOrGuid < 0: `guid` of the creature / game object / etc.
```
is enough.

Reply:
Just in case it's not clear. When you start working with smart_scripts, you don't know well how is the guid and entry topic, I thought it was interesting to extend it a little bit giving another point of view, beyond what was already written. So I added, "in other words"

Reply:
I highly disagree.

It is like saying a banana is a yellow fruit

In other words the banana has the color yellow

Reply:
I already deleted it in the commit.

Reply:
You are too fast D: XD